caddy.go

```diff
@@ -516,9 +493,48 @@ func startWithListenerFds(cdyfile Input, inst *Instance, restartFds map[string]r
 	return nil
 }
 
-func executeDirectives(inst *Instance, filename string,
-	directives []string, sblocks []caddyfile.ServerBlock) error {
+func ValidateAndExecuteDirectives(cdyfile Input, inst *Instance, justValidate bool) (*Instance, error) {
+
+	//If parsing only inst will be nil, create an instance for this function call only.
```

Comment:
Hmm, I'm surprised `go fmt` doesn't put a space between the `//` and `If`, but please do. :)